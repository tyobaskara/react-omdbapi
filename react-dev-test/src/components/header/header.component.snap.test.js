import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Header from './header.component';

test('Header snaptest', () => {
  const props = {};
  const component = renderer.create(
    <BrowserRouter>
      <Header {...props} />
    </BrowserRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});