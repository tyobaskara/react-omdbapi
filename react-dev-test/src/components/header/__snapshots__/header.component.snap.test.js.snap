// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snaptest 1`] = `
<div
  className="header"
>
  <a
    className="logo-container"
    href="/"
    onClick={[Function]}
  >
    <svg
      className="logo"
    >
      crown.svg
    </svg>
  </a>
</div>
`;
